### PR TITLE
Move slate prefab out of examples

### PR DESCRIPTION
**Pull request by julenka**
## Overview
Move Slate prefab out of Examples and into SDK/Features/UX/Slate. Update the HandInteractionPanZoomExample scene to be SlateExample scene.

## Changes
- Fixes: #4345 
- Remove unused DragEnabler Script
- Removed unused SlateEnabler script
- Update documentation to point to new paths
- Rename HandInteractionPanZoomExample to SlateExample scene

## Verification
Verified that the slate example works as expected.

### CODE REVIEW

File: Documentation/README_Slate.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ![Slate](../Documentation/Images/Slate/MRTK_Slate_Main.png)
 
-The [`Slate.prefab`](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.Examples/Demos/HandTracking/Prefabs/Slate.prefab) offers a thin window style control for displaying 2D content, for example plain text or articles including media. It offers a grabbable title bar as well as *Follow Me* and *Close* functionality. The content window can be scrolled via articulated hand input.
+The [`Slate.prefab`](../Assets/MixedRealityToolkit.SDK/Features/UX/Prefabs/Slate/Slate.prefab) offers a thin window style control for displaying 2D content, for example plain text or articles including media. It offers a grabbable title bar as well as *Follow Me* and *Close* functionality. The content window can be scrolled via articulated hand input.
```
Comment (keveleigh) [inline Documentation/README_Slate.md]: This relative path won't work on the github.io page, I believe.

**Comment by wiwei** [comment]
Note that others may have already started using SlateEnabler.cs and DragEnabler.cs - I have deleted things in the past that were unused in our code, but were used by others. The likelihood of this is probably low because we didn't have example usage or comments in the code about exactly how you'd use them, but still, a thing to be aware of.

I think it would be worth also updating this page:

https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/UpdatingToGA.html

To include information on this move.

**Pull request merged**